net-misc/rsync/rsync-3.2.3-r2.ebuild

```diff
@@ -54,13 +54,16 @@ python_check_deps() {
 	has_version "dev-python/commonmark[${PYTHON_USEDEP}]"
 }
 
+PATCHES=(
+	"${FILESDIR}/${PN}-3.2.3-glibc-lchmod.patch"
+	"${FILESDIR}/${PN}-3.2.3-glibc-lchmod-better-fix.patch"
+)
+
 src_prepare() {
 	default
-	if [[ "${PV}" == *9999 ]] ; then
-		eaclocal -I m4
-		eautoconf -o configure.sh
-		eautoheader && touch config.h.in
-	fi
+	eaclocal -I m4
+	eautoconf -o configure.sh
+	eautoheader && touch config.h.in
```

Comment:
We need to inherit autotools to call these functions. Currently, that only happens in the live ebuild.

Reply:
done